src/ontology/mondo-edit.obo

```diff
@@ -50456,21 +50468,26 @@ intersection_of: disease_has_location UBERON:0000473 ! testis
 id: MONDO:0003125
 name: testicular sex cord-stromal neoplasm
 def: "A benign or malignant tumor that arises from the testis. It is composed of granulosa cells, Leydig cells, Sertoli cells, and fibroblasts. Each of these cell types may constitute the only cellular component that is present in the tumor or it may be mixed with other cell types in various combinations. The prognosis can not always be predicted on histologic grounds alone. Approximately, ten percent of these tumors may metastasize. Representative examples include granulosa cell tumor, Leydig cell tumor, Sertoli cell tumor, and tumors of the thecoma-fibroma group." [NCIT:C6358]
+subset: ordo_disease {source="Orphanet:363489"}
 synonym: "sex cord-stromal neoplasm" RELATED [DOID:4757]
-synonym: "Sex cord-stromal neoplasm of testis" RELATED [NCIT:C6358]
-synonym: "Sex cord-stromal neoplasm of the testis" RELATED [NCIT:C6358]
-synonym: "Sex cord-stromal tumor of testis" EXACT [DOID:4757, NCIT:C6358]
-synonym: "sex cord-stromal tumor of testis" EXACT []
-synonym: "Sex cord-stromal tumor of the testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal neoplasm of testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal neoplasm of the testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal tumor of testis" EXACT [DOID:4757, NCIT:C6358, Orphanet:363489]
+synonym: "sex cord-stromal tumor of the testis" RELATED [NCIT:C6358]
```

Comment:
this is almost identical to the syn above, but this one is related and the other is exact. I think they both should be exact.